lib/arjdbc/vertica/adapter.rb

```diff
@@ -278,6 +278,16 @@ def vertica_random_temp_table_name
       "temporary_table_#{::SecureRandom.hex}"
     end
 
+    def type_to_sql(type, limit = nil, precision = nil, scale = nil) #:nodoc:
+      super unless type == :integer
+
+      if native = native_database_types[type.to_sym]
+        (native.is_a?(Hash) ? native[:name] : native).dup
```

Comment:
Since we're calling out `:integer` explicitly, would it make more sens to say something like:
```
native_database_types[:integer][:name].dup
```
?